From 334756fa12ff637889df2ebdc49db878f1e0da6d Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Fri, 16 May 2014 19:40:35 +0300
Subject: [PATCH 2/4] Move the footnote about the Config module to the first
 reference

The new Config reference was introduced in cc34dcc0 but the
footnote wasn't moved accordingly.
---
 perl-policy.sgml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/perl-policy.sgml b/perl-policy.sgml
index abe04d9..54e97d3 100644
--- a/perl-policy.sgml
+++ b/perl-policy.sgml
@@ -96,7 +96,8 @@
 	  <package>perlapi-<var>abiname</var></package> for all released
 	  package versions it is compatible with. The choice of
 	  <var>abiname</var> is arbitrary, but if it differs from
-	  <tt>$Config{version}</tt>, it must be specified in
+	  <tt>$Config{version}</tt><footnote>see the
+	  <tt>Config</tt> module</footnote>, it must be specified in
 	  <tt>$Config{debian_abi}</tt>.
 	</p>
       </sect>
@@ -149,8 +150,7 @@
 /usr/local/share/perl/<var>version</var>
 		</example>
 		Where <var>version</var> indicates the current Perl
-		version (<tt>$Config{version}</tt><footnote>see the
-		<tt>Config</tt> module</footnote>).
+		version (<tt>$Config{version}</tt>).
 	      </p>
 	    </item>
 	    <tag><var>vendor</var></tag>
-- 
2.0.0.rc2

